proto/planetmintgo/dao/distribution_order.proto

```diff
@@ -18,4 +16,10 @@ message DistributionOrder {
   int64 firstPop = 10; 
   int64 lastPop = 11;
   string proposer = 12;
+  string earlyInvestorAddr = 13; 
```

Comment:
above you called it `DistributionAddrEarlyInv` so here it should be `earlyInvAddr`